cla_backend/apps/legalaid/tests/test_views.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client
+from django.contrib.auth.models import Permission
+from django.core.urlresolvers import reverse
+
+from legalaid.views import FullCaseViewSet
+from cla_backend.apps.call_centre.permissions import *
+from cla_backend.urls import *
+from rest_framework import routers
+from legalaid.tests.views.test_base import CLAOperatorAuthBaseApiTestMixin
+
+
+class FullCaseViewSetTestCase(CLAOperatorAuthBaseApiTestMixin,TestCase):
+    def setUp(self):
+        super(FullCaseViewSetTestCase, self).setUp()
+        self.url = reverse('call_centre:case-list')
+
+    def test_filter_queryset_for_unicode_characters_status_code_200(self):
+        response = self.client.get(self.url+'?search=Mark%20O%E2%80%99Brien', HTTP_AUTHORIZATION='Bearer %s' % self.operator_manager_token)
```

Comment:
Either url encode or don't in this test and the following. Keep them consistent.

And generally include spaces around + for string concatenation. But we'll set up Black and pre-commit hooks.